tests/functional_tests/uri.py

```diff
@@ -142,16 +142,6 @@ def test_monero_uri(self):
         assert res.uri.recipient_name == utf8string[0]
         assert not 'unknown_parameters' in res or len(res.unknown_parameters) == 0
 
-        res = wallet.make_uri(address = address, recipient_name = utf8string[0], tx_description = utf8string[1], amount = 1000000000000, payment_id = '1' * 64)
```

Comment:
How about the following, to be able to test if the error is being detected:
```Python
try:
    res = wallet.make_uri(address = address, recipient_name = utf8string[0], tx_description = utf8string[1], amount = 1000000000000, payment_id = '1' * 64)
    except Exception:
       # 8067
       # make_uri only allows short payment id
       assert True
    else:
       # Exception not raised
      assert False

    # spaces must be encoded as %20
   (...)
```

Reply:
updated